src/log/file_appender.cpp

```diff
@@ -141,9 +160,11 @@ namespace fc {
       {
          fc::create_directories(my->cfg.filename.parent_path());
 
-         if(!my->cfg.rotate)
```

Comment:
I haven't walked all of the code, but if rotate == true, will removing this cause open be called twice (once here, and once in rotate_files()) ?